### PR TITLE
Stop functionality added to running validator

### PR DESCRIPTION
**Pull request by sammanadh**
1. Stop button added to stop a running validator container.
2. Logs now will only appear of the validator is running.

### CODE REVIEW

File: src/renderer/nav/Validator.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ const Validator = () => {
   return (
     <div className="row">
       {!(validator.status === NetStatus.Running) &&
-      !(validator.status === NetStatus.Starting) ? (
+        !(validator.status === NetStatus.Starting) ? (
```
Comment (SvenDowideit) [inline src/renderer/nav/Validator.tsx:55]: stylefight?

Comment (SvenDowideit) [reply src/renderer/nav/Validator.tsx]: exactly that :)

**Comment by SvenDowideit** [comment]
mmm, i think we should move the stop button to the top of the screen - when the user resizes the window they can get to a point where the stop button is hidden

this also suggests that the log isn't resizing right, but i presume that's not new

Screen Shot 2022-06-21 at 2 22 43 pm

**Review by SvenDowideit — APPROVED** [review]
yup LGTM - I feel weird about the run race condition, and the lack of feedback to the user, but that's for a future pr :)

**Comment by nathanleclaire** [comment]
Nice work Samman 👏

**Pull request merged**